### ISSUE TITLE
Import-Excel help is missing the -ImportColumns parameter

### ISSUE THREAD

**Issue by KeithALane** [dfinke/ImportExcel#1663]
I had a reason to use the **Import-Excel -ImportColumns** parameter today, and noticed that the parameter seems to be missing from the .xml help. 

I think the files that would need to change are: 
_mdHelp/en/import-excel.md
en/ImportExcel-help.xml_

There is an related example 
_Examples/ImportColumns/ImportColumns.ps1_

The sections that would need updating are 
**SYNTAX, PARAMETERS, EXAMPLES** 

I haven't worked on these type of help files with PlatyPS yet so I might try and figure out the process at some point.

**Comment by dfinke** [comment]
@KeithALane thanks for using ImportExcel. Correct, there is no help for that. Never got around to build a workflow for PlatyPS building the help.

Check out the examples - I believe there is one for Import Columns.

https://github.com/dfinke/ImportExcel/tree/master/Examples

**Comment by KeithALane** [comment]
I think I figured out the process with PlatyPS... it was a bit trickier than I was expecting. 
PlatyPS didn't like the ASCII art tables in the examples section and wouldn't validate unless I added a comment tag before them. 

When I got to GitHub it was detecting every line as a change which appeared to be due to the line widths changing when I ran PlatyPS's New-MarkdownHelp. So instead of that I manually marked up the changes in this branch. 
https://github.com/dfinke/ImportExcel/compare/master...KeithALane:ImportExcel:ImportColumns?expand=1

I'm not going to try and claim that what I changed is correct or even a good description, as this is the first time I've even attempted editing markdown/xml using PlatyPS. So rather than submitting a PR, I'd prefer to let you fix it in a way that is consistent with whatever practices you are using to manage the other help files. 

I also noticed the -Raw parameter also appears to be missing in the same way. 

Anyway, thanks for the great module, I use it every day!
